Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -122,47 +193,60 @@ bool CruRawReader::compareRDH(const o2::header::RDHAny* rdhPrev, const o2::heade
       LOG(warn) << "ERDH FEEID are not identical in rdh.";
       checkNoWarn();
     }
-    incrementErrors(TRDParsingBadRDHFEEID, 0, 0, 0, 0);
+    if (mOptions[TRDVerboseErrorsBit]) {
+      LOG(error) << "ERDH FEEID are not identical in rdh.";
+    }
+    incrementErrors(TRDParsingBadRDHFEEID);
     return false;
   }
   if (o2::raw::RDHUtils::getEndPointID(rdhPrev) != o2::raw::RDHUtils::getEndPointID(rdhCurr)) {
     if (mMaxWarnPrinted > 0) {
-      LOG(warn) << "ERDH  EndPointID are not identical in rdh.";
       checkNoWarn();
     }
-    incrementErrors(TRDParsingBadRDHEndPoint, 0, 0, 0, 0);
+    if (mOptions[TRDVerboseErrorsBit]) {
```

Comment:
Instead of adding everywhere the `mOptions[TRDVerboseErrorsBit]` plus a duplicate of the log message, could you not just change the `checkNoWarn()` and `checkNoErr()` methods to **not** decrement the mMaxErrsPrinted if the verbose bit is set? Would make the code much cleaner

Reply:
true